### ISSUE TITLE
While Using LibWrapper, Foundry 0.8.6, and the Forge I am getting Depreciation Errors

### ISSUE THREAD

**Issue by AdamantiteAdventurer** [ruipin/fvtt-lib-wrapper#39]
In the console view I am seeing the following issues, which seem to be adding a slight delay to the start up: 

- The Document#_id property is deprecated in favor of Document#id or Document#data#_id. Support will be removed in 0.9.0
- You are calling PlaceableObject#setFlag which has been deprecated in favor of Document#setFlag. Support will be removed in 0.9.0
- You are retrieving the value of CONFIG.Item.entityClass which has been renamed to CONFIG.Item.documentClass. Support for the old configuration value will be removed in 0.9.0
- PF2e | Failed to execute onBeforePrepareData on rule element [object Object]. TypeError: Cannot read property 'type' of null
- PF2E | Unknown rule element PF2E.RuleElement.FlatModifer

I also exported the console for the session:

[Console_Export_Forge_Foundry.log](https://github.com/ruipin/fvtt-lib-wrapper/files/6651479/Console_Export_Forge_Foundry.log)

**Comment by ruipin** [comment]
I have no idea why you opened an issue on this repository, none of those messages are even tangentially related to libWrapper.

Each of those is related to a module or system which hasn't been fully updated to support 0.8.x. You'll want to figure out which ones, and submit a bug on their repositories.

Closing as invalid.

**Comment by AdamantiteAdventurer** [comment]
I submitted this on libWrapper cause all of the error messages pointed to LibWrapper, so I was checking to see if this was the issue related to it directly. I am going through and systematically checking for latency. 

I am still looking into it to see which mods are causing the issue, however since you chose to word it that way (feels kinda aggressive) and said you have no idea why I opened an issue here or that the messages are even tangentially related work libWrapper, the error messages literally call out libWrapper: 


call_wrapped | @ | **libWrapper**-wrapper.js:466
-- | -- | --
&nbsp; | 🎁libWrapperInit | @ | **libWrapper**-api.js:592
&nbsp; | 🎁Game.prototype.initialize#0 | @ | **libWrapper**-wrapper.js:160



![2021-06-15 10_06_57-Foundry Virtual Tabletop • A Standalone Virtual Tabletop Application and 5 more ](https://user-images.githubusercontent.com/6942488/122067548-a96ff100-cdc1-11eb-8455-26b9a858651b.png)

Thanks for taking the time to reply and I hope that helps you have a better idea why I opened the issue on libWrapper first since every single error I posted literally and _tangentially_ called out libWrapper. Sorry for the incorrectly posted issue and for wasting you time.
